src/sass/partials/global/_mixins.scss

```diff
@@ -0,0 +1,126 @@
+@mixin button ($border-color, $color, $border-color-hover, $color-hover) {
+  background-color: setting-get('white');
+  border-color: $border-color;
+  border-radius: 45px;
+  border-style: solid;
+  border-width: setting-get('input border width');
+  color: $color;
+  padding: .5em 2em;
+
+  &:hover {
+    border-color: $border-color-hover;
+    color: $color-hover;
+  }
+}
+
+@mixin checkbox {
+  display: none;
+
+  + label {
+    position: relative;
+
+    &::before {
+      border: setting-get('input border width') solid setting-get('input border color');
+      content: '';
+      display: inline-block;
+      height: 1em;
+      margin-right: setting-get('rhythm') / 2;
+      transform: translateY(-.125em);
+      vertical-align: middle;
+      width: 1em;
+    }
+  }
+
+  &:checked + label::before {
+    background: setting-get('secondary color');
+    box-shadow: inset 0 0 0 setting-get('input border width') / 1.5 setting-get('white');
+  }
+}
+
+@mixin hyphens($hyphen-type) {
```

Comment:
We have [autoprefixer](https://github.com/postcss/autoprefixer). We do not need this mixin